lib/generators/spree_products_qa/install/install_generator.rb

```diff
@@ -1,29 +1,18 @@
 module SpreeProductsQa
   module Generators
     class InstallGenerator < Rails::Generators::Base
-
-      class_option :auto_run_migrations, :type => :boolean, :default => false
-
-      def add_javascripts
-        append_file 'vendor/assets/javascripts/spree/frontend/all.js', "//= require spree/frontend/spree_products_qa\n"
-        append_file 'vendor/assets/javascripts/spree/backend/all.js', "//= require spree/backend/spree_products_qa\n"
```

Comment:
we want to migrate to newer turbo based approach for spree frontend..
and remove sassc and older format of css and js..